### PR TITLE
fix performance report show, and add test for it

### PR DESCRIPTION
**Pull request by casaper**
# [Story in Trello](https://trello.com/c/ftZmgptG/96-bug-forgot-to-display-group-offer-performance-numbers-in-view)

### Done for approval?

* [ ] Acceptance criteria are met
* [ ] Code quality checks are green
* [ ] Readme updated if needed
* [ ] Story under test
* [ ] Mobile works
* [ ] Seeds created
* [ ] Translated
* [ ] Code is reviewed

### Done after the merge?

* [ ] Deployed to staging after the merge
* [ ] Tested manually on staging

### CODE REVIEW

File: app/views/performance_reports/show.html.slim [hunk 1]
```diff
@@ -37,7 +37,20 @@ table.table
       td
     tr.active
       td= t('.volunteers.active_total')
-      td= @report_content['global']['volunteers']['active']
+      td= @report_content['global']['clients']['active']
```
Comment (chrysalag) [inline app/views/performance_reports/show.html.slim]: Shouldn't then the above line change as well? ` td= t('.volunteers.active_total')`?

File: app/views/performance_reports/show.html.slim [hunk 2]
```diff
@@ -37,7 +37,20 @@ table.table
...
+
+    tr
+      td= t('.clients.total')
+      td= @report_content['global']['clients']['total']
+    tr
+      td= t('.clients.new')
+      td= @report_content['global']['clients']['new']
+    tr
+      td= t('.clients.resigned')
+      td
```
Comment (chrysalag) [inline app/views/performance_reports/show.html.slim]: Do we have info to display here?

File: app/views/performance_reports/show.html.slim [hunk 3]
```diff
@@ -82,6 +82,19 @@ table.table
...
 
     tr
+      td= t('.clients.total')
+      td= @report_content['zuerich']['clients']['total']
+    tr
+      td= t('.clients.new')
+      td= @report_content['zuerich']['clients']['new']
+    tr
+      td= t('.clients.resigned')
+      td
```
Comment (chrysalag) [inline app/views/performance_reports/show.html.slim]: Same here?

File: app/views/performance_reports/show.html.slim (hunk 2 above)
Comment (casaper) [reply app/views/performance_reports/show.html.slim]: thats what I was mentioning. Thats the missing number.

File: app/views/performance_reports/show.html.slim (hunk 1 above)
Comment (casaper) [reply app/views/performance_reports/show.html.slim]: right!

**Pull request merged**